website/docs/r/policy_asset_association.html.markdown

```diff
@@ -8,14 +8,14 @@ description: |-
 
 # incapsula_policy_asset_association
 
-Provides a Incapsula Policy Asset Association resource. 
+Provides an Incapsula Policy Asset Association resource. This resource enables you to apply policies to assets in your account.
 
 ## Example Usage
 
 ```hcl
 resource "incapsula_policy_asset_association" "example-policy-asset-association" {
-  policy_id  = incapsula_policy.example-policy.id
-  asset_id   = incapsula_site.example-site-dns.id 
+  policy_id  = "123456"
```

Comment:
The best practice of Terraform is to use a reference instead of an explicit ID, why changing it?